src/views/home/Home.tsx

```diff
@@ -36,7 +36,7 @@ const Home: React.FC = () => {
       <Grid container spacing={3} columns={{ sm: 6, md: 8, lg: 12 }} paddingBottom="2rem">
         <MenuCard title="About" text="Background Information." img="/images/GNS-home.png" url="/About" />
         <MenuCard title="Curves and Spectra" text="Site hazard and UHS plots." img="/images/SpectralAccelChart.png" url="/Hazardcurves" />
-        <MenuCard title="Hazard Maps" text="Hazard levels across NZ." img="images/hazard_map.jpeg" url="/HazardMaps" />
+        <MenuCard title="Hazard Maps" text="Hazard levels across NZ." img="images/hazard_map.png" url="/HazardMaps" />
```

Comment:
this link is now broken (too much search and replace?). Revert to jpeg link

Reply:
ah whoops nice spotting